ert_shared/main.py

```diff
@@ -210,6 +210,22 @@ def get_ert_parser(parser=None):
     es_mda_parser.set_defaults(func=run_cli)
     es_mda_parser.add_argument("config", type=valid_file, help=config_help)
 
+
+    workflow_description = "Executes the workflow given"
+    workflow_parser = subparsers.add_parser(
+        "workflow", help=workflow_description, description=workflow_description
+    )
+    workflow_parser.add_argument(
+        "--verbose", action="store_true", help="Show verbose output", default=False
+    )
+    workflow_parser.add_argument(
+        "--name", 
+        help="Name of workflow",
+        required=True
```

Comment:
Why don't we make this argument "anonymous", so that we don't need to specify the `--name`?
The call would then be something like `ert workflow my_fancy_workflow`

Reply:
I have no particular argument for or against positional arguments vs named arguments...

Reply:
I vote for @andreabrambilla's suggestion :)

Reply:
majority rules :)